Changes

```diff
@@ -547,6 +547,13 @@ Working version
   (Sébastien Hinderer, review by Antonin Décimo, David Allsopp and Florian
   Angeletti)
 
+* #12751: --with-target-bindir configure option implemented. This option
+  refers to the location of the runtime binaries on the target system for a
+  cross-compiler and is embedded in executables produced by ocamlc. It does
+  not affect the bytecode executables installed as part of the build. The old
+  old mechanism `make TARGET_BINDIR=..` no longer works.
```

Comment:
```suggestion
  not affect the bytecode executables installed as part of the build.
  The old mechanism `make TARGET_BINDIR=..` no longer works.
```

Reply:
Ah, I reveal that I forgot how to configure my editor to reflow text automatically... 🙈